taipy/common/config/ScenarioConfig.py

```diff
@@ -0,0 +1,46 @@
+import matplotlib.pyplot as plt
```

Comment:
This must be an optional dependency. Please verify it is installed before using it.